discovery-provider/src/queries/query_helpers.py

```diff
@@ -829,10 +829,12 @@ def get_karma(session, ids, time=None, is_playlist=False, xf=False):
         ).subquery()
 
     query = (
-        session.query(saves_and_reposts.c.item_id, func.count(Follow.followee_user_id))
+        session.query(
+            saves_and_reposts.c.item_id,
+            cast(func.sum(AggregateUser.follower_count), Integer),
```

Comment:
For my own understanding: why is this cast necessary?

Reply:
It returns the sum as a sql double instance which don't interop w/ normal python math operations